internal/api/api.go

```diff
@@ -25,16 +26,9 @@ type Client interface {
 	// variables.
 	NewQuery(query string) Request
 
-	// NewRequest creates a GraphQL request.
+	// NewRequest creates a GraphQL request with gzip compression turned on.
```

Comment:
I actually wouldn't change this comment — this can be an implementation detail now.